front/app/containers/Admin/projects/all/new/Projects/Table/constants.ts

```diff
@@ -0,0 +1,9 @@
+// TODO: make this configurable state instead of a constant.
```

Comment:
Given that we also still need to add a project manager column, and that people don't seem to care about the `projectStart` and `projectEnd` columns, I decided to hide them like this. We can later see if we want to maybe make this configurable state instead of a constant, depending on what comes out of the designs.